repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
+          "default": ["slave_public"],
```

Comment:
I agree with @jdef that the default "slave_public" is not a good idea for security reasons. The solution is **not** to open up the apiserver via public slaves, but to implement authz and auth. We have a ticket in Jira and for security reasons this should be a priority soon.